packages/api-v4/src/index.ts

```diff
@@ -0,0 +1,118 @@
+import { app } from "@azure/functions";
+import { getUsers, getUserById, postUsers } from "./functions/users";
+import { getPaymentById, getPayments, postPayment } from "./functions/payments";
+import { getReservationById, getReservations, patchReservationById } from "./functions/reservations";
+import { getFavorites, postFavorites, deleteFavorite } from "./functions/favorites";
+import { getListingById, getListings } from "./functions/listings";
+import { openApi } from "./functions/openapi";
+import { postCheckout } from "./functions/checkout";
+
+//#region User Function
+app.get("get-users", {
```

Comment:
I love this `index.ts` file aggregating the bindings from the functions. It loosens the coupling between the endpoints and the Azure Function infrastructure a bit.

It also makes it easier to see the possibility of generating functions or bindings based on convention. I've always wished we could Railsify the creation of Azure Functions.